app/views/followships/update.js.erb

```diff
@@ -0,0 +1,10 @@
+alert("here is")
+$('.btn').bind('ajax:success',function() {
+
+  
+  $(this).closest('.row').fadeOut()
+  $('.Followers').append("<%= j render(:partial => 'row') %>")
+
```

Comment:
Please name your partial to something more related to module and appropriate one.

Reply:
Done